### PR TITLE
[3.3] Scrollable Tabs

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: components/tabstrip/scrollable-tabs.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+To scroll the tabs only, set the `Scrollable` bool parameter of the TabStrip to `true` (defaults to `false`). You should also set the `Width` (for tabs with [Position]({%slug tabstrip-tabs-position%}) set to top and bottom) and `Height` (for tabs with [Position]({%slug tabstrip-tabs-position%}) set to left and right) to achieve the desired behavior.
+
+>caption Scroll the tabs only
+
+<div class="skip-repl"></div>
+````HorizontalScrolling
+<TelerikTabStrip Scrollable="true"
+                 Width="300px"
+                 TabPosition="Telerik.Blazor.TabPosition.Top">
+    <TabStripTab Title="First">
```
Comment (dimodi) [inline components/tabstrip/scrollable-tabs.md:24]: Generate the tabs with a loop to make the example simpler.

File: components/tabstrip/scrollable-tabs.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+title: Scrollable Tabs
+page_title: TabStrip Scrollable Tabs
+description: Scrollable Tabs in the TabStrip for Blazor.
+slug: tabstrip-scroll-tabs
```
Comment (dimodi) [inline components/tabstrip/scrollable-tabs.md:5]: scroll-tabs may be harder to remember and use without checking. Why not use scrollable-tabs, as the article title?

File: components/tabstrip/scrollable-tabs.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+tags: telerik,blazor,tab,strip,tabstrip,scroll,tabs
+published: True
+position: 15
+---
+
+# TabStrip Scrollable Tabs
+
+By default, if the accumulative width or height of the tabs in the TabStrip exceed the width or height of the browser a browser-level scrollbar will appear.
```
Comment (dimodi) [inline components/tabstrip/scrollable-tabs.md:13]: Consider a simpler version. It's "cumulative" by the way.
```suggestion
If the TabStrip dimensions exceed the width or height of the page, a browser-level scrollbar will appear. 
```

File: components/tabstrip/scrollable-tabs.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+tags: telerik,blazor,tab,strip,tabstrip,scroll,tabs
+published: True
+position: 15
+---
+
+# TabStrip Scrollable Tabs
+
+By default, if the accumulative width or height of the tabs in the TabStrip exceed the width or height of the browser a browser-level scrollbar will appear. 
+
+To scroll the tabs only, set the `Scrollable` bool parameter of the TabStrip to `true` (defaults to `false`). You should also set the `Width` (for tabs with [Position]({%slug tabstrip-tabs-position%}) set to top and bottom) and `Height` (for tabs with [Position]({%slug tabstrip-tabs-position%}) set to left and right) to achieve the desired behavior.
```
Comment (dimodi) [inline components/tabstrip/scrollable-tabs.md:15]: Can be a list of 3 steps as well.

File: components/tabstrip/overview.md [hunk 5]
```diff
@@ -13,18 +13,18 @@ position: 0
 This article provides information about the <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> and its core features.
 
 >caption In this article:
-* [Basics](#basics)
+* [Creating Blazor TabStrip](#creating-blazor-tabstrip)
 * [Features](#features)
```
Comment (dimodi) [inline components/tabstrip/overview.md:17]: We should list the features themselves and remove the Features heading. See /wiki/Docs-Overview-Page-Template in the private repo.

**Review by dimodi — CHANGES_REQUESTED** [review]
Consider these -
* article slug's grammar to match the article's title, for easier usage
* generate the tabs with a loop to make the code snippets short and simple
* replace the Features heading with the actual features - https://github.com/telerik/blazor/wiki/Docs-Overview-Page-Template

**Review by jivanova — APPROVED** [review]

**Pull request merged**